### PR TITLE
🌟Training - Adding Presenter component

### LINKED ISSUE #943: Training - Add Presenter component
URL: https://www.ssw.com.au/ssw/Events/Training/Angular-Superpowers-Tour.aspx

**Description:**
Need to develop a component for training pages to add presenters block.

Image
**Figure: Presenter component on existing site**

### PR DESCRIPTION
**Pull request by amankumarrr**
Fixes #943 

URL: `/training/internship-fullstack`

**New Presenter component screenshots:**

image

**Figure: Presenters in desktop view**

image

**Figure: Presenters in mobile/iPad view**

### CODE REVIEW

File: components/training/presenterBlock.tsx [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+      )}
+      {!data.otherEvent && <div className="pb-12"></div>}
+    </Container>
+  );
+};
+
+const PresenterCard = ({ presenter, schema, index }) => {
+  return (
+    <div
+      className="flex h-fit flex-col rounded-md border-b-4 border-b-sswRed bg-gray-100 p-10 text-center text-xl shadow drop-shadow md:h-full"
```
Comment (jaydenalchin) [inline components/training/presenterBlock.tsx]: The cards shouldn't have a drop shadow - everything else is looking great!

**Review by jaydenalchin — CHANGES_REQUESTED** [review]

File: components/training/presenterBlock.tsx (hunk 1 above)
Comment (amankumarrr) [reply components/training/presenterBlock.tsx]: I removed the drop-shadow from the div.

**Review by bradystroud — APPROVED** [review]
LGTM

File: components/training/presenterBlock.tsx [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+      )}
+      {!data.otherEvent && <div className="pb-12"></div>}
+    </Container>
+  );
+};
+
+const PresenterCard = ({ presenter, schema, index }) => {
+  return (
+    <div
+      className="flex h-fit flex-col rounded-md border-b-4 border-b-sswRed bg-gray-100 p-10 text-center text-xl shadow md:h-full"
```
Comment (jaydenalchin) [inline components/training/presenterBlock.tsx]:
```suggestion
      className="flex h-fit flex-col rounded-md border-b-4 border-b-sswRed bg-gray-100 p-10 text-center text-xl md:h-full"
```

**Review by jaydenalchin — CHANGES_REQUESTED** [review]
Sneaky unwanted shadow class remains

**Review by jaydenalchin — APPROVED** [review]
LGTM 🚀

**Pull request merged**